parsl/dataflow/strategy.py

```diff
@@ -177,7 +177,15 @@ def _strategy_simple(self, tasks, *args, kind=None, **kwargs):
             # FIXME probably more of this logic should be moved to the provider
             min_blocks = executor.provider.min_blocks
             max_blocks = executor.provider.max_blocks
-            tasks_per_node = executor.provider.tasks_per_node
+            if isinstance(executor, IPyParallelExecutor):
+                tasks_per_node = executor.workers_per_node
+            elif isinstance(executor, HighThroughputExecutor):
```

Comment:
Yikes, this is unfortunate, we should open an issue

Reply:
Added a note on #530